### PR TITLE
Return keys from generate().

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Addresses issue https://github.com/digitalbazaar/did-method-key-js/issues/19.

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+package-lock=false
```
Comment (davidlehn) [inline .npmrc:1]: We haven't usually been checking this sort of thing in, in favor of people setting that in their global config.  Was this added by mistake?

Comment (dmitrizagidulin) [reply .npmrc:1]: @davidlehn np, I can remove it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # did:key driver ChangeLog
 
+## 0.6.1 - TBD
+
+### Changed
+- Return public/private key pair from `generate()`, available on `didDoc.keys`.
+  Example: `const didDoc = await driver.generate(); didDoc.keys // -> keys obj`.
```
Comment (mattcollier) [inline CHANGELOG.md:7]: I'm -1 to putting code in the changelog like this, also you updated the readme.
```suggestion
```

Comment (dmitrizagidulin) [reply CHANGELOG.md:7]: @mattcollier no prob, removing.

**Pull request merged**